curriculum/challenges/english/07-scientific-computing-with-python/learn-string-manipulation-by-building-a-cipher/666064e915eba7aa1de03f6b.md

```diff
@@ -0,0 +1,48 @@
+---
+id: 666064e915eba7aa1de03f6b
+title: Step 51
+challengeType: 20
+dashedName: step-51
+---
+
+# --description--
+
+Now, let's fix the error by removing the line that tries to print the `alphabet` variable outside of the `caesar` function.
+
+Remove the line `print(alphabet)` at the end of your code.
```

Comment:
```suggestion
```
It seems a bit redundant to me.

Reply:
I think it makes it clear what the camper should do, which helps prevent any confusion about which line needs to be removed.

Reply:
you already said that in the line above tho, this seems more like a hint, and I see there is an hint almost identical, so I would say we are good without this line

Reply:
Okay that makes sense